src/TraceEvent/TraceEvent.cs

```diff
@@ -242,9 +242,9 @@ public DateTime SessionEndTime
         {
             get
             {
-                var ret = QPCTimeToDateTimeUTC(sessionEndTimeQPC);
+                var ret = QPCTimeToDateTimeUTC(sessionEndTimeQPC).ToLocalTime();
```

Comment:
:memo: I generally prefer to use `DateTimeOffset` instead of `DateTime` to avoid these kinds of problems, but that would be an API change instead of just a bug fix.